### ISSUE TITLE
Bad video stream on Argus 3 Pro

### ISSUE THREAD

**Issue by Veuchez** [thirtythreeforty/neolink#279]
**Describe the bug**
On Argus 3 Pro both in mainStream and in subStream the video stream is bad, slow, jerky, sometimes accelerated ..
The cameras are connected in 5GHz and the coverage is excellent.

**Versions**
NVR software: VLC
Neolink software: The last one
Reolink camera model and firmware: Argus 3 Pro Firmware 3.0.0.907_22042201

**Comment by Veuchez** [comment]
Is this issue always related to timestamps and UDP ports?
Were you able to find a solution?
Thanks!

**Comment by QuantumEntangledAndy** [comment]
I suspect so. Have you tried just monitoring the substream? It tends to help with this.

**Comment by Veuchez** [comment]
Yes I tried, it gets a little better, it's always a bad stream ...
